draft-ietf-quic-http.md

```diff
@@ -275,11 +275,12 @@ fully closed.
 A server can send a complete response prior to the client sending an entire
 request if the response does not depend on any portion of the request that has
 not been sent and received. When this is true, a server MAY request that the
-client abort transmission of a request without error by sending a RST_STREAM
-with an error code of NO_ERROR after sending a complete response and closing its
-stream. Clients MUST NOT discard responses as a result of receiving such a
-RST_STREAM, though clients can always discard responses at their discretion for
-other reasons.
+client abort transmission of a request without error by triggering a QUIC
+STOP_SENDING with error code HTTP_EARLY_RESPONSE, sending a complete response,
+and cleanly closing its streams. Clients MUST NOT discard complete responses as
+a result of receiving a STOP_SENDING, though clients can always discard
```

Comment:
HTTP clients won't receive a STOP_SENDING, since that's a QUIC frame (I realize that this was similar earlier with RST_STREAM)... perhaps something like "as a result of receiving a solicited stream reset"? I don't have better ideas.